src/final2/subtests/InvalidCommandTest.java

```diff
@@ -155,17 +155,11 @@ public void createExistingAntTest() {
 	@Test
 	public void invalidAntNameTest() {
 		for (String invalidAntName : INVALID_ANT_NAMES) {
-			inputFile = new String[] {
-					"0000",
-					"0000",
-					"0a00",
-					"0000"
-			};
 			runs = new Run[] {
 					new ErrorRun("create " + invalidAntName + ",1,1"),
 					quit()
 			};
-			sessionTest(runs, Input.getFile(inputFile));
+			sessionTest(runs, Input.getFile(TASK_SHEET_INPUT_FILE_1));

```

Comment:
I do not agree on using TASK_SHEET_INPUT_FILE_1 here:
TASK_SHEET_INPUT_FILE_1 should only be used, when the playing field is not important. Here it is important that the cell 1,1 is empty and not an obstacle. (In the other examples the palying field is not important at all...)